enemy-test/Makefile

```diff
@@ -0,0 +1,54 @@
+# pod = Pineapple of Doom
+PROG = pod_enemy
+
+export SDCCDIR=~/sdcc  # might differ
+GBDK_2020 = ../gbdk-2020/
+CC = $(GBDK_2020)/build/gbdk/bin/lcc -Wa-l -Wl-m -Wl-j
+IMG2GB = img2gb
+
+_OBJ = $(patsubst %.c,%.o,$(wildcard src/*.c))
+
+all: build
+
+build: $(PROG).gb
+
+debug: $(wildcard src/*.c)
+	$(CC) -S $^
+
+$(PROG).gb: $(_OBJ)
+	$(CC) -o $(PROG).gb $^
+
+%.o: %.c
+	$(CC) -o $@ -c $<
+
+assets:
+	$(IMG2GB) tileset \
+		--output-c-file=src/dummy-tileset.c \
+		--output-header-file=src/dummy-tileset.h \

```

Comment:
FIXME: The output files are wrong, should be `dummy.sprites.c` (according to your code)